tests/Feature/Api/V1/AchievementSetClaimTest.php

```diff
@@ -43,11 +44,16 @@ public function testGetUserClaims(): void
         $system = System::factory()->create();
         /** @var Game $game */
         $game = Game::factory()->create(['ConsoleID' => $system->ID]);
-        /** @var AchievementSetClaim $claim */
-        $claim = AchievementSetClaim::factory()->create([
-            'GameID' => $game->ID,
-            'User' => $this->user->User,
-        ]);
+
+        insertClaim(
+            $this->user->User,
+            $game->ID,
+            ClaimType::Primary,
+            ClaimSetType::NewSet,
+            ClaimSpecial::None,
+            Permissions::Developer
+        );
+        $claim = AchievementSetClaim::first();
```

Comment:
Testing the `insertClaim()` function specifically was not the goal here, the `AchievementSetClaim::factory()` would suffice, however, since I refactored this method already i guess it should stay like this for now until the public request files get their own tests.